### DISCUSSION TITLE
RDPro autodiscovered hosts on LAN inconsistent connections

### DISCUSSION THREAD

**Discussion by tastle73** [rustdesk/rustdesk#8798] [Q&A]
I am using the latest release of RD Pro.

All my endpoints are in a LAN together with no firewalls in play.

 In my client I see all of the other installed endpoints, but I am not able to connect to all of them some of them time out and I have no idea why.  The far client is up, logged in and RD is running as a service.   We are all connected to the same server via the API.  

I have nothing to go with this.

**Comment by rustdesk** [comment]
If the RustDesk server is installed on intranet, please check this, https://github.com/rustdesk/rustdesk/wiki/FAQ#deploy-rustdesk-server-in-intranet

And also please check this, https://github.com/rustdesk/rustdesk/wiki/FAQ#connection-reset-by-the-peer
